src/utils/dateUtils.ts

```diff
@@ -0,0 +1,12 @@
+const ONE_WEEK_MILLIS = 7 * 24 * 60 * 60 * 1000;
+
+export const calculateWeeksSinceDate = (date: Date): number => {
+  const now = new Date();
+  return getWeeksBetween(new Date(date), now);
+};
+
+export const getWeeksBetween = (date1: Date, date2: Date): number => {
+  return Math.round(
+    Math.abs(date1.getTime() - date2.getTime()) / ONE_WEEK_MILLIS
+  );
+};
```

Comment:
Her er uker basert på antall 7-dagere mellom. Vi hadde ikke noe dato-bibliotek i denne appen (dayjs eller moment), men mulig vi vil vurdere å ta inn noe sånt? Datoer er alltid vanskelig.

Reply:
Gjerne dato-bibliotek for min del, tid er vanskelig ja.

Reply:
MomentJS tror jeg er i ferd med å bli deprecated, men gjerne dayjs som vi bruker i syfomodiaperson f.eks.